src/components/Dropdown/Dropdown.st.css

```
@@ -95,7 +95,16 @@
 }
 
 .dropdown:content-visible .dropdownBase::arrowIcon {
-    transform: rotate(180deg);
+    transform: translateY(-50%) rotate(180deg);
+}
+
+.dropdown .dropdownBase::arrowIcon {
```

Looks great, awesome work!

Note that this won't be applied to the mobile dropdown (which is rendered with a `select` element).
To fix this, I think you need to just add the correct selector, like this:
```suggestion
.dropdown .dropdownBase::arrowIcon, 
.dropdown .dropdownNativeSelect::arrowIcon {
```
(see here: https://github.com/wix/wix-ui-tpa/pull/324/files#diff-c6f15d870b6a55bc847324c839142cceR70)